packages/flutter/lib/src/painting/gradient.dart

```diff
@@ -507,6 +507,7 @@ class LinearGradient extends Gradient {
       colors: interpolated.colors,
       stops: interpolated.stops,
       tileMode: t < 0.5 ? a.tileMode : b.tileMode, // TODO(ianh): interpolate tile mode
+      transform: t < 0.5 ? a.transform : b.transform,
```

Comment:
not a big deal, but if you feel like it, you could remove the TODO on the line above (or alternatively, add one on the new line, with a link to https://github.com/flutter/flutter/issues/443)

(same for all three gradients)